test/jdk/java/awt/Cursor/CursorDragTest/ListDragCursor.java

```diff
@@ -78,7 +76,61 @@ public static Frame createUI() {
         panel.add(list);
 
         frame.add(panel);
-        frame.setBounds(300, 100, 300, 150);
-        return frame;
+        frame.setSize(300, 150);
+        frame.setLocation(450, 400);
+        frame.setVisible(true);
+    }
+
+    static void createInstructionsFrame() {
+        String INSTRUCTIONS = """
```

Comment:
Since it is local var (non-static & non-final var)

```suggestion
        String instructions = """
```